src/main/java/frc/robot/subsystems/drivetrain/CTREDrivetrainIO.java

```diff
@@ -33,15 +39,26 @@ public class CTREDrivetrainIO implements DrivetrainIO {
     private final PIDController yController = new PIDController(10.0, 0.0, 0.0);
     private final PIDController headingController = new PIDController(7.5, 0.0, 0.0);
 
-
     @SafeVarargs
     public CTREDrivetrainIO(
+            double robotWeightKG,
+            double robotMOI,
             SwerveDrivetrainConstants drivetrainConstants,
             SwerveModuleConstants<TalonFXConfiguration, TalonFXConfiguration, CANcoderConfiguration>... moduleConstants
     ) {
         this.drivetrain = new SwerveDrivetrain<>(
                 TalonFX::new, TalonFX::new, CANcoder::new, drivetrainConstants, moduleConstants
         );
+        RobotConfig robotConfig;
+        ModuleConfig moduleConfig;
+
+        //TODO FIX!!!
+        moduleConfig = new ModuleConfig(Constants.MK4_WHEEL_RADIUS, 5.0,
+                Constants.WHEEL_COF, DCMotor.getKrakenX60Foc(1), moduleConstants[0].DriveMotorGearRatio,
+                moduleConstants[0].DriveMotorInitialConfigs.CurrentLimits.StatorCurrentLimit, 4);
+
+        robotConfig = new RobotConfig(robotWeightKG, robotMOI, moduleConfig, drivetrain.getModuleLocations());
```

Comment:
There's no reason to have separate variable declaration and assignment here, right?